pgusinit/pgusinit.c

```diff
@@ -216,47 +217,47 @@ int main(int argc, char* argv[]) {
     banner();
     int i = 1;
     while (i < argc) {
-        if (strcmp(argv[i], "/?") == 0) {
-            usage();
+        if (stricmp(argv[i], "/?") == 0) {
+            usage(argv[0]);
             return 0;
-        } else if (strcmp(argv[i], "/a") == 0) {
+        } else if (stricmp(argv[i], "/a") == 0) {
             if (i + 1 >= argc) {
-                usage();
+                usage(argv[0]);
                 return 255;
             }
             e = sscanf(argv[++i], "%hu", &buffer_size);
             if (e != 1 || buffer_size < 8 || buffer_size > 256) {
-                usage();
+                usage(argv[0]);
                 return 3;
             }
-        } else if (strcmp(argv[i], "/d") == 0) {
+        } else if (stricmp(argv[i], "/d") == 0) {
             if (i + 1 >= argc) {
-                usage();
+                usage(argv[0]);
                 return 255;
             }
             e = sscanf(argv[++i], "%hu", &dma_interval);
             if (e != 1 || dma_interval > 256) {

```

Comment:
Unlike buffer_size, dma_interval cannot have a value of 256 - it gets "outb"ed to the port as 0, so 255 is the max.

Reply:
did the requested changes